api/prometheus/v1/api_test.go

```diff
@@ -259,7 +279,7 @@ func TestAPIs(t *testing.T) {
 			reqPath:   "/api/v1/query",
 			reqParam: url.Values{
 				"query":   []string{"2"},
-				"time":    []string{testTime.Format(time.RFC3339Nano)},
```

Comment:
Tests formatting to currently fail with the following error message:

```
unexpected request parameters: want map[query:[2] time:[1651799194.268618]], got map[query:[2] time:[2022-05-06T02:06:34.268618+01:00]]
```

so I've updated the tests to use `formatTime` for the time being.